examples/react-native/storybook/stories/Authenticator.stories.tsx

```diff
@@ -147,7 +147,9 @@ const signUpProps = {
 const verifyUserProps = {
   ...sharedProps,
   ...getComponentSlots(Authenticator.VerifyUser),
-  fields: [],
+  fields: [
+    { name: 'email', type: 'radio' as const, value: 'jeff@example.com' },
```

Comment:
as const?

Reply:
It's because of the TS settings in the sample app